.papr.sh

```diff
@@ -30,35 +32,41 @@ pip install -r requirements.txt
 
 PAPR_INVENTORY=${PAPR_INVENTORY:-.papr.inventory}
 PAPR_RUN_UPDATE=${PAPR_RUN_UPDATE:-0}
+PAPR_EXTRAVARS=${PAPR_EXTRAVARS:-}
+
+# Human-readable output
+export ANSIBLE_STDOUT_CALLBACK=debug
 
 # ping the nodes to check they're responding and register their ostree versions
-ansible -vvv -i $PAPR_INVENTORY nodes -a 'rpm-ostree status'
+ansible -v -i $PAPR_INVENTORY nodes -a 'rpm-ostree status'
```

Comment:
Recommend using `-vv` as it will give the yml file and line number for each task which significantly aids in troubleshooting when tasks are not named well or use variables.

Reply:
Fixed for all playbook invocations, keeping `-v` for rpm-ostree status, as we don't really care about this info